resources/views/app/includes/contents/main.blade.php

```diff
@@ -0,0 +1,125 @@
+<div>
+    <h4>SUYABAY RELOADED</h4>
+        <div class="row">
+
+            <div class="card col s12 m4">
+
+                <div class="card-image waves-effect waves-block waves-light">
+                    <img clss = "activator" src="http://a.fastcompany.net/multisite_files/fastcompany/imagecache/slideshow_large/slideshow/2015/05/3046442-slide-s-2-this-startup-pays-african-students.jpg">
+                </div>
+
+                <div class="card-content">
+                    <span class="card-title activator grey-text text-darken-4">Suya Andela
+                        <i class="material-icons right">view_headline</i>
+                    </span>
+                </div>
+
+                <div class="card-reveal">
+                    <span class="card-title grey-text text-darken-4">Card Title<i class="material-icons right">close</i></span>
+                        <p>Here is some more information about this product that is only revealed once clicked on.</p>
```

Comment:
The paragraph tag doesn't need to be shifted like this, it should still be the on the same grid line with the span tag. Please Fix